enclave_manager/Dockerfile

```diff
@@ -14,80 +14,65 @@
 # ------------------------------------------------------------------------------
 
 # Description:
-#   Builds the environment needed to build Trusted Compute Framework.
+#   Builds the environment needed to build Avalon SGX Enclave manager.
 #
 #  Configuration (build) parameters
 #  - proxy configuration: https_proxy http_proxy ftp_proxy
 #  - sgx mode:
 #
-# Build:
-#   $ docker build docker -f docker/Dockerfile.tcf-dev -t tcf-dev
-#   if behind a proxy, you might want to add also below options
-#   --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy --build-arg ftp_proxy=$ftp_proxy
-#
-# Run:
-#   $ cd TrustedComputeFramework
-#   $ docker run -v $(pwd):/project/TrustedComputeFramework -it tcf-dev /bin/bash
-#   (to run with SGX HW, add options '--device=/dev/isgx -v /var/run/aesmd:/var/run/aesmd ')
-#   make clean && make
-
+# -------------=== enclave manager build ===-------------
 
-FROM ubuntu:bionic
+FROM ubuntu:bionic as base_image
 
-# Ignore timezone prompt in apt
 ENV DEBIAN_FRONTEND=noninteractive
 
-# Add necessary packages
 RUN apt-get update \
  && apt-get install -y -q \
     build-essential \
     ca-certificates \
-    cmake \
-    git \
-    make \
     pkg-config \
-    cmake \
-    protobuf-compiler \
-    libprotobuf-dev \
-    python3-dev \
-    virtualenv \
-    python3-pip \
     python3-toml \
     python3-requests \
     python3-colorlog \
     python3-twisted \
+ && apt-get -y -q upgrade \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
```

Comment:
build-essential and  pkg-config could be moved to build_image

Reply:
Done